swap_meet/electronics.py

```diff
@@ -1,2 +1,19 @@
-class Electronics:
-    pass
+from swap_meet.item import Item
+class Electronics(Item):
+    """Child class that inherits attributes and methods from Item class as parent class:"""
+    
+
+    def __init__(self, condition = 0, age = 0):
+        """Setting category attribute as default with "Electronics" and inherits attributes from Item class."""
+        super().__init__(category= "Electronics", condition = condition, age = age)
```

Comment:
Tiny style point here and elsewhere: when defining default arguments don't include spaces on either side of the equals sign. We would instead write it like:
```
def __init__(self, condition=0, age=0):
    super().__init__(category="Electronics", condition=condition, age=age)
```